APE24.rst

```diff
@@ -0,0 +1,507 @@
+******************************************
+Astropy Affiliated Packages with pyOpenSci
+******************************************
+
+author: Pey Lian Lim, Leah Wasser, William Jamieson, Derek Homeier, Moritz Günther
+
+date-created: 2023 July 13
+
+date-last-revised: 2023 December 1
+
+date-accepted: TBD
+
+type: Process
+
+status: Discussion
+
+.. NOTE: Most of this stuff is from https://docs.google.com/document/d/1hIl3y6nrcepv-tKKQLVB4u0a3fXACQ1Qrg0Y6VrJykg/view
+
+
+Abstract
+========
+
+Since the early days of the Astropy Project, we had recognized the need
+to accommodate and developed an internal process for acknowledging 
```

Comment:
```suggestion
to accommodate and develop an internal process for acknowledging 
```

Reply:
Was a bit unclear what was originally meant, but I suppose I was also a little bit unclear, possibly omitting a (non-Oxford) comma in my suggestion https://github.com/astropy/astropy-APEs/pull/87#discussion_r1388757260 – my understanding being that the internal process has been developed since. It's both in the past now anyways....